### PR TITLE
OHARA-1332 Allow users to start and stop a FTP sink connector

### PR DESCRIPTION
**Pull request by eechih**

### CODE REVIEW

File: ohara-manager/client/src/components/pages/PipelinePage/connectors/__tests__/JdbcSource.test.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ const props = {
   updateHasChanges: jest.fn(),
   updateGraph: jest.fn(),
   loadGraph: jest.fn(),
+  graph: [],
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/PipelinePage/connectors/__tests__/JdbcSource.test.js]: I've fixed this in #21 :)

Comment (eechih) [reply ohara-manager/client/src/components/pages/PipelinePage/connectors/__tests__/JdbcSource.test.js]: Got it. I've reset this commit.

**Comment by konekoya** [comment]
retry manager

**Comment by konekoya** [comment]
retry manager

**Comment by konekoya** [comment]
@eechih Could you rebase this PR? Thanks :)

**Comment by konekoya** [comment]
Looks good to me. Thanks, @eechih

**Pull request merged**